.github/workflows/CI.yml

```
@@ -1,31 +1,31 @@
 name: CI
-on: [ push, pull_request ]
+on: [push, pull_request]
 jobs:
   Build:
     strategy:
       matrix:
-        os: [ ubuntu-latest ]
-        node-version: [ 14 ]
+        os: [ubuntu-latest]
+        node-version: [22]
```

```suggestion
        node-version: [20]
```
cf. https://camunda.slack.com/archives/GP70M0J6M/p1743412897483559?thread_ts=1743412206.752249&cid=GP70M0J6M